### DISCUSSION TITLE
Suggestions: close all tabs

### DISCUSSION THREAD

**Discussion by jcfaria** [sumatrapdfreader/sumatrapdf#4012] [Ideas]
Sumatra is my favorite PDF visualizer for a lot of years! I am grateful for the continuous development.

I miss an option in the popmenu of the tabs to close all the tabs.

J.C.Faria

**Comment by DrNup** [comment]
That would be very useful and probably easy to implement !?

**Comment by GitHubRulesOK** [comment]
- Ensure using  a recent pre-release (This month) [https://www.sumatrapdfreader.org/prerelease](https://www.sumatrapdfreader.org/prerelease)

- Add your [own shortcut as per manual](https://www.sumatrapdfreader.org/docs/Customizing-keyboard-shortcuts) with `CloseAllTabs` as the action

Otherwise Right click the leftmost tab (not home) close ALL to right then close that one you used to close the others

Updated per @DrNup comment
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/0a77e8c1-ce85-47e5-8010-e451bdb1ac45)

**Comment by DrNup** [comment]
Sumatra PDF Reader v3.6. pre-release build version 16066 now has a menu entry for closing all tabs and it works perfectly !
Many thanks to the developer(s)
Regards
